### PR TITLE
Issue 139 test duplicate files

### LINKED ISSUE #139: RavenFramework Tests should check for duplicate files listed
--------
Issue Description
--------
##### What did you expect to see happen?
RavenFramework tests should fail if a file is listed in multiple places.

##### What did you see instead?
The tests sometimes crash because os.remove is called multiple times on the same file if the exists check is slow.

##### Please attach the input file(s) that generate this error.  The simpler the input, the faster we can find the issue.
```
+ [./melcorInterfaceForwardSamplingNoExe]
+   type = 'RavenFramework'
+   input = 'test_melcor_interface.xml'
+   output = 'MelcorInterface/dumpMelcorPointSet.csv MelcorInterface/dumpMelcorHistorySet_2.csv MelcorInterface/dumpMelcorHistorySet.csv'
+   csv = 'MelcorInterface/dumpMelcorPointSet.csv'
+   text = 'MelcorInterface/testMelcor/1/test.i'
+   test_interface_only = True
+   rel_err = 0.0001
+ [../]
```

----------------
For Change Control Board: Issue Review
----------------
This review should occur before any development is performed as a response to this issue.
- [ ] 1. Is it tagged with a type: defect or improvement?
- [ ] 2. Is it tagged with a priority: critical, normal or minor?
- [ ] 3. If it will impact requirements or requirements tests, is it tagged with requirements?
- [ ] 4. If it is a defect, can it cause wrong results for users? If so an email needs to be sent to the users.
- [ ] 5. Is a rationale provided? (Such as explaining why the improvement is needed or why current code is wrong.)

-------
For Change Control Board: Issue Closure
-------
This review should occur when the issue is imminently going to be closed.
- [ ] 1. If the issue is a defect, is the defect fixed?
- [ ] 2. If the issue is a defect, is the defect tested for in the regression test system?  (If not explain why not.)
- [ ] 3. If the issue can impact users, has an email to the users group been written (the email should specify if the defect impacts stable or master)?
- [ ] 4. If the issue is a defect, does it impact the latest stable branch? If yes, is there any issue tagged with stable (create if needed)?
- [ ] 5. If the issue is being closed without a merge request, has an explanation of why it is being closed been provided?

### PR DESCRIPTION
**Pull request by joshua-cogliati-inl**
--------
Pull Request Description
--------
##### What issue does this change request address? 
#139 

Closes #139

##### What are the significant changes in functionality due to this change request?
This now checks for duplicate entries in the testing.

I don't like that it skips the test instead of failing the test, but checkRunnable can't cause a test to fail.  (Setting status to bucket_fail is treated as bucket_silent)

----------------
For Change Control Board: Change Request Review
----------------
The following review must be completed by an authorized member of the Change Control Board.
- [x] 1. Review all computer code.
- [x] 2. If any changes occur to the input syntax, there must be an accompanying change to the user manual and xsd schema. If the input syntax change deprecates existing input files, a conversion script needs to be added (see Conversion Scripts).
- [x] 3. Make sure the Python code and commenting standards are respected (camelBack, etc.) - See on the [wiki](https://github.com/idaholab/raven/wiki/RAVEN-Code-Standards#python) for details.
- [x] 4. Automated Tests should pass, including run_tests, pylint, manual building and xsd tests. If there are changes to Simulation.py or JobHandler.py the qsub tests must pass.
- [x] 5. If significant functionality is added, there must  be tests added to check this. Tests should cover all possible options.  Multiple short tests are preferred over one large test. If new development on the internal JobHandler parallel system is performed, a cluster test must be added setting, in <RunInfo> XML block, the node ```<internalParallel>``` to True.
- [x] 6. If the change modifies or adds a requirement or a requirement based test case, the Change Control Board's Chair or designee also needs to approve the change.  The requirements and the requirements test shall be in sync.
- [x] 7. The merge request must reference an issue.  If the issue is closed, the issue close checklist shall be done.
- [x] 8. If an analytic test is changed/added is the the analytic documentation updated/added?

### CODE REVIEW

File: tests/framework/pca_rom/tests [hunk 1]
```diff
@@ -10,24 +10,24 @@
  [./MVNAbs]
   type = 'RavenFramework'
   input = 'test_MVN_abs.xml'
-  output = 'MVNAbs/testPrintPointSetAbs_dump.csv MVNAbs/testPrintPointSetAbs_dump.csv'
-  csv = 'MVNAbs/testPrintPointSetAbs_dump.csv MVNAbs/testPrintPointSetAbs_dump.csv'
-  xml = 'MVNAbs/testPrintPointSetAbs_dump.xml MVNAbs/testPrintPointSetAbs_dump.xml'
+  output = ''
```
Comment (alfoa) [inline tests/framework/pca_rom/tests]: does it need to be empty or it can be just removed?

Comment (joshua-cogliati-inl) [inline tests/framework/pca_rom/tests]: It can be removed.  I removed the empty output lines.

**Comment by alfoa** [comment]
----------------
For Change Control Board: Change Request Review
----------------
The following review must be completed by an authorized member of the Change Control Board.
- [ ] 1. Review all computer code. ***n/a. Just additional check on the test system.***
- [ ] 2. If any changes occur to the input syntax, there must be an accompanying change to the user manual and xsd schema. If the input syntax change deprecates existing input files, a conversion script needs to be added (see Conversion Scripts).  ***n/a. Just additional check on the test system.***
- [ ] 3. Make sure the Python code and commenting standards are respected (camelBack, etc.) - See on the [wiki](https://github.com/idaholab/raven/wiki/RAVEN-Code-Standards#python) for details.  ***n/a. Just additional check on the test system.***
- [ ] 4. Automated Tests should pass, including run_tests, pylint, manual building and xsd tests. If there are changes to Simulation.py or JobHandler.py the qsub tests must pass. ***n/a. Just additional check on the test system.Anyhow the tests pass***
- [ ] 5. If significant functionality is added, there must  be tests added to check this. Tests should cover all possible options.  Multiple short tests are preferred over one large test. If new development on the internal JobHandler parallel system is performed, a cluster test must be added setting, in <RunInfo> XML block, the node ```<internalParallel>``` to True. ***n/a. Just additional check on the test system.***
- [ ] 6. If the change modifies or adds a requirement or a requirement based test case, the Change Control Board's Chair or designee also needs to approve the change.  The requirements and the requirements test shall be in sync. ***n/a. Just additional check on the test system.***
- [ ] 7. The merge request must reference an issue.  If the issue is closed, the issue close checklist shall be done.  ***#139***
- [ ] 8. If an analytic test is changed/added is the the analytic documentation updated/added? ***n/a. Just additional check on the test system.***

**Comment by alfoa** [comment]
Check list passed.

Merging.

**Pull request merged**